### ISSUE TITLE
Avoid sending JSON:API version on payload

### ISSUE THREAD

**Issue by marcemira** [paknahad/jsonapi-bundle#46]
Is there a way to configure the package to opt out of sending the jsonapi spec version on each response?
```
"jsonapi": { "version":"1.0" }
```
Thanks

**Comment by paknahad** [comment]
No, unfortunately, there is no configuration.
in each document, you need return null for "getJsonApi".

**Comment by marcemira** [comment]
Yep yep, seems like it's how `woohoolabs/yin` works, thanks @paknahad :grin:

**Comment by paknahad** [comment]
I would appreciate any PR that makes configurable any part.
